packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -8,7 +8,7 @@ import Map from 'babel-runtime/core-js/map';
 
 import {InvalidEmailAddressError} from './ediscovery-error';
 
-const defaultOptions = {timeoutMs: 10000}; // 10s default timeout
+const defaultOptions = {timeoutMs: 30000}; // 30s default timeout
```

Comment:
Default timeout increased as >10s latency was seen for content and content summary APIs